posthog/temporal/usage_reports/run_usage_reports.py

```diff
@@ -172,8 +148,13 @@ def async_get_instance_metadata(p):
 
         print(f"Sending usage reports {total_orgs} orgs")  # noqa: T201
 
-        for org_report in inputs.org_reports.values():
+        org_count = 0
+        for org_report in org_reports.values():
             try:
+                org_count += 1
+                if org_count % 500 == 0:
+                    print(f"Processed {org_count}/{total_orgs} organizations...")  # noqa: T201
```

Comment:
Any reason we are using print so much instead of logging? I'm fine with ignoring warnings with `# noqa`, but I'd generally have a reason.

Reply:
Logs aren't showing in the pods so it's the only way I can debug.

Reply:
Strange, we log all the time in other workflows without problems.

Reply:
How are you trying to log?

Reply:
I've tried so many things and gave up wasting time on it. 

https://posthog.slack.com/archives/C0113360FFV/p1742327851631179. 

Happy to come back, remove these and try again after getting this workflow working.

Reply:
Fair enough :+1: